site/docs/patterns/split-button.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Split button
+layout: DetailPattern
+aliases:
+  - /salt/patterns/split-button
+data:
+  components: ["Button", "Menu", "Segmented Button Group", "Tooltip"]
+---
+
+A split button enables users to perform a main action and access relevant, supplementary actions inside a menu. 
+
+<Diagram
+  src="/img/patterns/split-button/split-button-hero-shot.png"
+  alt="Example showing a split button."
+  border
+/>
+
+## When to use
+
+Use the pattern for actions with multiple associated commands. Users are able to immediately perform the default action and access supplementary actions, which include more specific or less commonly-used actions.
+
+For example, a paste-focused split button could include: 
+- Default command: Paste 
+- Alternative commands: Keep source formatting, Match formatting, Paste special...
+
+## How to build
+
+### Anatomy
+
+A split button contains:
+
+1. **Main action:** the default action a user can take.
+2. **Arrow button:** contains a chevron-down icon and triggers the menu.
+3. **Supplementary actions:** lower priority actions displayed in a menu. 
+
+<Diagram
+  src="/img/patterns/split-button/split-button-anatomy.png"
+  alt="Split button showing supplementary actions."
+  border
+/>
+
+### Layout
+
+- Use the segmented button group component as the base structure of the split button.
+- Use the chevron-down icon for the menu trigger.
+- Use a menu component to display supplementary actions for users to take.
+
+By default, the menu displays below the arrow button.
```

Comment:
```suggestion
```
as per conversation below this isn't by default but something a reader should do